src/AMSlib/ml/uq.hpp

```diff
@@ -142,6 +140,23 @@ class UQ
     }
   }
 
+  void updateModel(std::string model_path, std::string uq_path = "")
```

Comment:
uq_path is unused?

Reply:
For FAISS we need 2 paths. Cause those are encoded by separate entities. For DeltaUQ the model includes UQ.